department-of-veterans-affairs/caseflow
# Case Details - Base Page

Resolves #6874 

### Description
Implement flow between Assign Hearings and Case Details. Hearings coordinator can click on veteran link in Assign Hearings to be taken to Case Details to review the appeal and confirm the hearing schedule assignment.

In this PR when the coordinator selects the Assign Hearing action in Case Details the task is completed in the back-end. The modal functionality will be part of the PR associated with issue #7072 .

### Acceptance Criteria
- Test case for ScheduleHearingTask passes.

### Testing Plan
1. Go to Assign Hearings and click on new link for veteran.
2. Confirm the CaseDetails page is rendered and that it has a task in the top section.
3. Confirm that when you select the Assign Hearing action the Mark Complete task modal displays.
4. Click on Complete Task and confirm the database task row is set to completed.
